docs/manuals/source/index.rst

```diff
@@ -5,6 +5,7 @@ For information regarding the newest release see the :ref:`release notes <bareos
 
 .. toctree::
 
+   /BareosBook.rst
```

Comment:
Maybe we need to discuss the information architecture we want for the documentation.
While we shouldn't change too much of the existing structure (people won't find things anymore) it probably makes sense to thoroughly think about where new things should go.